### PR TITLE
Ball speed penalties

### PR DESCRIPTION
**Pull request by rhololkeolke**
No stoppage.

Goals with kicks exceeding limit do not count. Increase the foul counter.

See https://docs.google.com/spreadsheets/d/1LfNNrE4ld0IkRGXSkKSKOnPDipdKbuH-InZJY2kATs4/edit?usp=sharing

### CODE REVIEW

File: chapters/offenses.adoc [hunk 1]
```diff
@@ -40,6 +48,8 @@ NOTE: Dribblers can still be used to dribble large distances with the ball as lo
 ==== Ball Speed
 A robot must not accelerate the ball faster than 6.5 meters per second in 3D space.
 
+Violations do not stop game play. Goals scored with kicks exceeding the ball speed limit will not be counted. Increase <<Fouls, foul counter>> by one.
```
Comment (MathewMacDougall) [inline chapters/offenses.adoc:51]: How is gameplay restarted after a goal is not counted?

**Review by MathewMacDougall — CHANGES_REQUESTED** [review]

File: chapters/offenses.adoc (hunk 1 above)
Comment (g3force) [reply chapters/offenses.adoc:51]: Good point. It is defined in the "Scoring goals" sections for other invalid goals: It is counted as if the ball crossed the goal line outside of the goal. A reference to this sections should probably be sufficient, but I haven't checked the formulations in detail.

Comment (g3force) [inline chapters/offenses.adoc:51]: We should use the grace period here as well. Making a fast pass and than scoring a goal should also not be allowed.

**Comment by rhololkeolke** [comment]
Closed in favor of #32

**Pull request closed without merging**